### ISSUE TITLE
Enable for source.jsx scope

### ISSUE THREAD

**Issue by bcomnes** [SublimeLinter/SublimeLinter-eslint#307]
ST4 ships with a new JSX syntax highlighter, and has a new source.jsx scope.  However, when this scope is used, eslint is disabled it seems.  What would be the best way to enable eslint on the source.jsx scope?

**Comment by bcomnes** [comment]
I added this to my sublime linter config and it worked:

```
// SublimeLinter Settings - User
{
  "linters": {
    "eslint": {
      "selector": "source.jsx, source.js - meta.attribute-with-value"
    }
  }
}
```

It would probably be a good idea to to add source.jsx to the default set from what I can tell.

**Comment by kaste** [comment]
Fix released as 4.2.2

**Comment by bcomnes** [comment]
Cool thank you @kaste !
